lib/events/filelog.go

```diff
@@ -532,6 +541,31 @@ func (l *FileLog) findInFile(fn string, query url.Values, total *int, limit int)
 	return retval, nil
 }
 
+// FilterOutSessionEnd returns whether to filter out session end event
+// determined by whether the input slice contains session end event session ID
+func FilterOutSessionEnd(ef EventFields, r []string) bool {
```

Comment:
Your mileage may vary: It took me a while to work out the out that the slice `r` is a list of session IDs. Perhaps rename it `sessionIDs`?